### ISSUE TITLE
Jelang Ramadhan Ribuan Botol Miras Dimusnahkan Polrestabes Surabaya

### ISSUE THREAD

**Issue by halodunia** [halodunia/halodunia#52]
Jelang Ramadhan Ribuan Botol Miras Dimusnahkan Polrestabes Surabaya<br>
http://ift.tt/22zFTru<br><p style="float: left;"><img alt="img" class="img-center" src="http://ift.tt/1XjSWOF" style="float: left; padding-right: 30px;" width="500px"></p>
<p><strong><em>Halo Dunia.Net,Surabaya</em></strong>.Dalam rangka menyambut datangnya Bulan Suci Ramadhan 1437 H, Polrestabes Surabaya menggelar pemusnahan barang bukti miras yang didapat dari operasi dan razia yang dilaksanakan dalam 5 hari terakhir.</p>
<p>Pelaksanaan operasi dan razia dengan sasaran miras ini sesuai dengan hasil rapat koordinasi Forkopimda Kota Surabaya pada Kamis (26/5) di Graha Polisi Istimewa Polrestabes Surabaya.</p>
<p>Polrestabes Surabaya memusnahkan miras berbagai jenis sejumlah 10.348 Botol dan Minuman Arak/ Oplosan Sebanyak 27 Jerigen. Tidak semua hasil operasi dimusnahkan namun ada sekitar 1.970 botol miras merk luar negeri yang masih dalam proses penyidikan.</p>
<p>Tidak hanya Kegiatan Pemusnahan Barang Bukti ini dihadiri oleh Forkopimda Kota Surabaya dan para tokoh agama serta toko masyarakat. turut hadir Mewakili Kapolda Jatim yakni Pejabat Utama Polda Jatim,Kamis (2/6/16).</p>
<p>Pelaksanaan Operasi dan Razia yang dilaksanakan hanya dalam 5 hari terakhir ini mendapat apresiasi dari Kapolrestabes Surabaya Kombespol Iman Sumantri yang disampaikan dalam sambutannya sebelum pelaksanaan Pemusnahan barang bukti minuman keras.</p>
<p>Setelah mendengar laporan dari Kasat Reskoba Polrestabes Surabaya AKBP Donny Adityawarman selaku Ketua Panitia. Pelaksanaan Pemusnahan dilaksanakan secara simbolis oleh Walikota Surabaya, Kapolrestabes Surabaya dan Danrem 084 BJ, selanjutnya pemusnahan dilaksanakan oleh seluruh undangan yang hadir.</p>
<p>Setelah seluruh undangan memusnahkan miras secara simbolis, Pemusnahan dilanjutkan oleh Alat Berat (Double Drum Roller) hingga seluruh barang bukti miras tidak tersisa lagi.</p>
<p>Ditemui usai memimpin pemusnahan, Kapolrestabes Surabaya mengungkapkan Pelaksanaan Pemusnahan barang bukti miras ini dilakukan sebagai upaya memberantas peredaran miras di Kota Surabaya menjelang datangnya bulan suci Ramadhan yang sebentar lagi akan dilaksanakan.</p>
<p>Kapolrestabes Surabaya memberikan apresiasi setinggi-tingginya kepada Masyarakat Kota Surabaya atas peran aktifnya dalam memberikan informasi terkait adanya peredaran minuman keras kepada pihak Kepolisian.</p>
<p>Harapan beliau tidak ada lagi korban dari masyarakat akibat minum minuman keras, sehingga masyarakat Kota Surabaya dapat melaksanakan Ibadah Puasa dengan Khusuk hingga bertemu dengan hari datangnya Kemenangan.</p>
<p>Diujung sambutannya Kombespol Iman Sumantri mengajak seluruh masyarakat untuk memerangi Pe redaran Miras Ilegal dan Oplosan dimulai dari diri kita, keluarga kita agar terhindar dari pengaruh negatif yang diakibatkan oleh Minuman Keras. <strong><em>(red/dw-1)</em></strong></p>
<p>&nbsp;</p>
<br><br>
via Kopidev News Feed http://ift.tt/1X0Y3Tv<br>
June 02, 2016 at 05:42PM
